pyinjective/core/broadcaster.py

```diff
@@ -12,6 +13,9 @@
 from pyinjective.constant import GAS_PRICE
 from pyinjective.core.gas_limit_estimator import GasLimitEstimator
 from pyinjective.core.network import Network
+from pyinjective.exceptions import BannedAddressError, OfacListFetchError
+
+OFAC_LIST_URL = "https://raw.githubusercontent.com/InjectiveLabs/injective-lists/master/wallets/ofac.json"
```

Comment:
As per Bojan's suggestion, I think the intention is fot the SDK to have a local copy of the `ofac.json` and read the local file when looking for the addresses. And we should have a helper method (could be a class method) to download an updated copy of the file, that we could use before releasing new SDK versions or the user cuould use tu update the list in their environment.

Reply:
It will be manually triggered:
a) By us when releasing new versions
b) By the users whenever they want

Reply:
Implemented this. Now `ofac.json` is in the root of the project and could be refreshed in cases a) and b) by running `python3 pyinjective/ofac.py`